jjb/dynamic/uperf.yml

```diff
@@ -0,0 +1,105 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+        # Disable logging
+        set +x
+        if [[ $SKIP_TLS_VERIFICATION ]]; then
+          oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL} --insecure-skip-tls-verify
+        else
+          oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
+        fi
+        # Run tests
+        set -x
+        pushd workloads/network-perf
+        ./run_hostnetwork_network_test_fromgit.sh
+        ./run_pod_network_test_fromgit.sh
+        ./run_serviceip_network_test_fromgit.sh
```

Comment:
@mohit-sheth @jtaleric Should we call the network_compare script to determine pass/fail here or maybe we can add it to the hostnetwork/pod_network/serviceip network test scripts in plow?

Reply:
IMHO leave the compare outside of this. 

https://github.com/cloud-bulldozer/plow/blob/master/workloads/network-perf/run_network_compare.sh

Should help with this.

Reply:
We need to also pass a "baseline_uperf_uuid"